src/components/select/CountryMultiSelectDropdown.tsx

```diff
@@ -0,0 +1,149 @@
+
+export type CountryMultiSelectDropdownProps<
+  TCountryCode extends string = string,
+> = CountryMultiSelectExpandedProps<TCountryCode> & {
+  isOpen?: boolean;
+};
+
+export const CountryMultiSelectDropdown = <
+  TCountryCode extends string = string,
+>({
+  className,
+  countries,
+  isOpen = false,
+  onChange,
+  selectedCountries,
+  style,
+}: CountryMultiSelectDropdownProps<TCountryCode>) => {
+  const [searchText, setSearchText] = useState('');
+
+  const { floatingStyles, refs, setVisible, visibility, visible, wrapperRef } =
+    useDropdown<HTMLDivElement, HTMLDivElement>({
+      offsetCrossAxis: 0,
+      offsetMainAxis: 5,
+      startOpen: isOpen,
+    });
+
+  const searchCountries = useCallback(
+    (searchTerm: string) => {
+      const { fuzzySearch } = getFuzzySearch({
+        array: countries,
+        options: {
+          keys: ['label', 'code'],
+          threshold: 0.1,
+        },
+      });
+
+      return fuzzySearch.search(searchTerm);
+    },
+    [countries],
+  );
+
+  const shownCountries: ICountryMultiSelectExpandedOption<TCountryCode>[] =
+    useMemo(() => {
+      if (searchText === '') {
+        return countries;
+      }
+
+      const results = searchCountries(searchText);
+      return results.map(x => x.item);
+    }, [countries, searchCountries, searchText]);
+
+  const renderSelected = () => {
+    if (!selectedCountries.length) {
+      return null;
+    }
+
+    return (
+      <div className={styles.prefixContainer}>
+        {selectedCountries.map(country => (
+          <button
+            className={styles.prefixTag}
+            onClick={() =>
+              onChange(selectedCountries.filter(x => x.code !== country.code))
+            }
+            type="button"
+          >
+            {country.icon}
+            <p>{country.label}</p>
+            <RemoveIcon size={20} />
+          </button>
+        ))}
+      </div>
+    );
+  };
+
+  const renderLabel = () => {
+    if (!selectedCountries.length) {
+      return 'Select countries';
+    }
+
+    return `Select countries (${selectedCountries.length} selected)`;
+  };
+
+  return (
+    <div ref={wrapperRef} className={className} style={style}>
+      <div ref={refs.setReference}>
+        <Input
+          className={styles.styledInput}
+          inputPrefix={renderSelected()}
+          inputSuffix="dog"
+          label={renderLabel()}
+          onChange={e => setSearchText(e.target.value)}
+          onFocus={() => setVisible(true)}
+          suffix={
+            <div className={styles.suffixActions}>
+              {!!selectedCountries.length && (
+                <Button
+                  icon={<RemoveCircleIcon />}
+                  onClick={() => onChange([])}
+                  variant="text"
+                />
+              )}
+              <button onClick={() => setVisible(true)} type="button">
+                <ChevronUpIcon size={20} />
+                <ChevronDownIcon size={20} />
```

Comment:
I'm confused about the functionality of the chevron up and down here. It's not a number input, and its' functionality is to show or hide the dropdown 🤔 
<img width="147" alt="image" src="https://github.com/Zonos/amino/assets/17950626/260e9eed-41aa-4dbc-a12e-294da92e3432">

Reply:
I just wanted to make it look like the current CountrySelect as much as possible, which has those icons. I will change the trigger events to be more like the other one as well.

Reply:
I see, the gap between two icons are what confused me. If you can style those two icons little bit closer like the old country select so that it will not look the same as the number input we are currently having.
<img width="44" alt="image" src="https://github.com/Zonos/amino/assets/17950626/076ebe1d-9192-4db2-8fc0-3d38b5736f86">